src/lib.jl

```diff
@@ -244,6 +244,38 @@ function sinkhorn2(μ, ν, C, ε; kwargs...)
     return pot.sinkhorn2(μ, ν, PyCall.PyReverseDims(permutedims(C)), ε; kwargs...)
 end
 
+"""
+    empirical_sinkhorn_divergence(X_s, X_t, reg; kwargs...)
+
+Compute the Sinkhorn Divergence from empirical data, where
+`X_s` and `X_t` are arrays representing samples in the source domain and target domain, respectively,
+and  `reg` is the regularization term. `a` and `b` are optinal sample weights in the source
+and target domain, respectively.
```

Comment:
This should be updated accordingly. `a` and `b` should not be explained, they are keyword arguments.